src/main/java/com/epam/aidial/core/log/GfLogStore.java

```diff
@@ -199,4 +216,108 @@ private static String formatTimestamp(long timestamp) {
         return LocalDateTime.ofInstant(Instant.ofEpochMilli(timestamp), ZoneId.of("UTC"))
                 .format(DateTimeFormatter.ISO_DATE_TIME);
     }
+
+    static String assembleStreamingResponse(Buffer response) {
+        try (Scanner scanner = new Scanner(new ByteBufInputStream(response.getByteBuf()))) {
+            StringBuilder content = new StringBuilder();
+            ObjectNode result = ProxyUtil.MAPPER.createObjectNode();
+            ObjectNode last = null;
+            ObjectNode choice = ProxyUtil.MAPPER.createObjectNode();
+            ObjectNode message = ProxyUtil.MAPPER.createObjectNode();
+            choice.set("message", message);
+            JsonNode usage = null;
+            JsonNode statistics = null;
+            JsonNode systemFingerprint = null;
+            JsonNode model = null;
+            scanner.useDelimiter("\n*data: *");
```

Comment:
it worth to add a comment for delimiter pattern as well

Reply:
fixed